src/include/duckdb/execution/reservoir_sample.hpp

```diff
@@ -24,20 +24,23 @@ class BaseReservoirSampling {
 
 	void SetNextEntry();
 
-	void ReplaceElement();
-
+	void ReplaceElement(double with_weight = -1);
 	//! The random generator
 	RandomEngine random;
 	//! Priority queue of [random element, index] for each of the elements in the sample
 	std::priority_queue<std::pair<double, idx_t>> reservoir_weights;
 	//! The next element to sample
-	idx_t next_index;
+	idx_t next_index_to_sample;
```

Comment:
@Maxxen I've changed some of the names here. I imagine this breaks serialization? Is there a test I can run to check if I did? And is there a document explaining how I can fix it?

Reply:
This class isn't getting serialized so this shouldn't break serialization. Also outside of enums we don't look at the code directly for serialization so changing variable names will not break the serialization format in any case.